src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -92,6 +98,7 @@
     // for JAXB
     @SuppressWarnings("unused")
     private Configuration() {
+
```

Comment:
Just a simple formatting issue: Remove this blank line from the empty method body, please.